.github/workflows/audit.yml

```diff
@@ -0,0 +1,19 @@
+name: Audit
```

Comment:
Cool! I did not know about audit workflows in Rust. This got me looking into it.